uaclient/api/u/pro/unattended_upgrades/status/v1.py

```diff
@@ -0,0 +1,195 @@
+import datetime
+import os
+from typing import Dict, List, Optional, Tuple, Union
+
+from uaclient import exceptions, messages
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.api.errors import APIError
+from uaclient.apt import get_apt_config_keys, get_apt_config_values
+from uaclient.config import UAConfig
+from uaclient.data_types import (
+    BoolDataValue,
+    DataObject,
+    DatetimeDataValue,
+    Field,
+    IntDataValue,
+    StringDataValue,
+)
+from uaclient.system import get_systemd_job_state
+
+UNATTENDED_UPGRADES_CONFIG_KEYS = [
+    "APT::Periodic::Enable",
+    "APT::Periodic::Update-Package-Lists",
+    "APT::Periodic::Unattended-Upgrade",
+    "Unattended-Upgrade::Allowed-Origins",
+]
+
+UNATTENDED_UPGRADES_STAMP_PATH = "/var/lib/apt/periodic/upgrade-stamp"
+
+
+class UnattendedUpgradesError(APIError):
+    pass
+
+
+class UnattendedUpgradesResult(DataObject, AdditionalInfo):
```

Comment:
I think this should include "Status" in the name
```suggestion
class UnattendedUpgradesStatusResult(DataObject, AdditionalInfo):
```